app/services/BoardSolvedService.coffee

```diff
@@ -0,0 +1,13 @@
+$ = require 'jquery'
+
+class BoardSolvedService
+
+  isBoardCleared: (boardBottomRow) ->
+    for value in boardBottomRow
+      return false unless value is ' '
+    return true
+
+  newGameBoard: ->
+    $( "#new-game-button" ).trigger( "click" );
```

Comment:
Shouldn't trigger `'click'`, but should call the method that is triggered by the new game button.  I think.